how-to/rotate/github-apps.rst

```diff
@@ -0,0 +1,22 @@
+Rotate Github App Private Keys
+==============================
+
+Releng uses Github Apps for authentication in a variety of places. This page
+outlines how to rotate the private keys associated with a Github App that are
+used to `generate an app installation token`_.
+
+Steps to Rotate
+---------------
+
+1. Open https://github.com/organizations/mozilla-releng/settings/apps and click
+   ``Edit`` next to the app you are rotating.
+2. Scroll down to the ``Private Keys`` section and click the ``Generate a
+   private key`` button. This will prompt a download of the private key and
+   associate the public key with the app.
```

Comment:
Is SecOps OK with generating the keys in the browser? (I presume it's done client side, but this seems like it may open up more or at least new surface area than generating a keypair with openssl.)

Reply:
Afaict, there's no option other than to do it in the browser. Github implicitly stores the public key, and provides the private key for download. You can't generate a key and then submit the public key to Github (at least it's not possible via the UI and I don't see any API endpoints for it either.. which makes sense because this key is needed to access the API).

Reply:
OK, just wanted to check!